disable otheropts menu and systemd support code, it conflicts with our changes

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/themes/openSUSE/src/common.inc
+++ b/themes/openSUSE/src/common.inc
@@ -385,12 +385,6 @@
     } { pop } ifelse
   } if
 
-  xmenu.otheropts {
-    otheropts.options xmenu.otheropts .xm_current get get dup "" ne {
-      "%s " bc.cmd dup length add sprintf
-    } { pop } ifelse
-  } if
-
   /cmdline.hidden over dup length .inp_hidden ge { .inp_hidden get } { pop .undef } ifelse def
 
   cmdline.hidden {
@@ -1703,8 +1697,6 @@ config.livecd {
 
 /config.product        "product"             "openSUSE" gfxconfig.set.str
 
-/config.init           "init"                        "" gfxconfig.set.str
-
 /serial.line0          "serial.line0"               [ ] gfxconfig.set.array_str
 /serial.line1          "serial.line1"               [ ] gfxconfig.set.array_str
 /serial.line2          "serial.line2"               [ ] gfxconfig.set.array_str
--- a/themes/openSUSE/src/gfxboot.cfg
+++ b/themes/openSUSE/src/gfxboot.cfg
@@ -150,7 +150,7 @@ spl=1
 autodown=0
 ; F-key assignments
 ; value can be one of: lang, video, install, kernelopts, dud, bits, keymap,
-; profile, restore, otheropts
+; profile, restore
 ; note1: 'bits' is only shown on x86_64, 'profile' only when a file 'profile'
 ; exists, 'restore' only when there is a boot entry 'restore'
 ; note2: F1 is always help, F9 enables speech output, F10 exits gfxboot
@@ -179,14 +179,12 @@ restore.title=
 key.F2=lang
 key.F3=profile
 key.F4=keymap
-key.F5=otheropts
 key.F11=restore
 ; add 'lang' option with current locale
 addopt.lang=0
 ; add 'keytable' option with current keymap
 addopt.keytable=0
-; default init program, default: /sbin/init
-init=/bin/systemd
+
 
 [live]
 ; set livecd flag
--- a/themes/openSUSE/src/main.bc
+++ b/themes/openSUSE/src/main.bc
@@ -23,7 +23,6 @@
 %% include dia_keymap.inc
 %% include dia_restore.inc
 %% include dia_net.inc
-%% include dia_otheropts.inc
 %% include panel.inc
 %% include keytables.inc
 %% include locale.inc
--- a/themes/openSUSE/src/panel.inc
+++ b/themes/openSUSE/src/panel.inc
@@ -167,7 +167,6 @@
   dup "video"      eq { pop [ /panel.video /panel.video.width /panel.video.update /video.init ] return } if
   dup "install"    eq { pop [ /panel.install /panel.install.width /panel.install.update /install.init ] return } if
   dup "kernelopts" eq { pop [ /panel.kernelopts /panel.kernelopts.width /panel.kernelopts.update /kernelopts.init ] return } if
-  dup "otheropts"  eq { pop [ /panel.otheropts /panel.otheropts.width /panel.otheropts.update /otheropts.init ] return } if
   dup "dud"        eq { pop [ /panel.dud /panel.dud.width /panel.dud.update /dud.init ] return } if
   dup "keymap"     eq { pop [ /panel.keymap /panel.keymap.width /panel.keymap.update /keymap.init ] return } if
 
